lib/LLVMPasses/ARCEntryPointBuilder.h

```diff
@@ -241,7 +241,7 @@ class ARCEntryPointBuilder {
                            isNonAtomic(OrigI) ? "swift_nonatomic_release"
                                               : "swift_release",
                            DefaultCC, RuntimeAvailability::AlwaysAvailable,
-                           {VoidTy}, {ObjectPtrTy}, {NoUnwind});
+                           {VoidTy}, {ObjectPtrTy}, {NoUnwind}, {});
```

Comment:
I think because we are starting off with memoryeffects::none and adding (or’ing) entries of the array MEMEFFECTS we want to pass MemoryEffects::unknown instead of the empty array here